src/assets/style.css

```diff
@@ -0,0 +1,245 @@
+  line-height: 48px;
+  margin-left: auto;
+  margin-right: auto;
+}
+.wizard--text {
+  width: 100%;
+  opacity: 0.56;
+  color: #211952;
+  font-family: Rubik;
+  font-size: 16px;
+  letter-spacing: 0.4px;
+  line-height: 24px;
+  margin-left: auto;
+  margin-right: auto;
+}
+.wizard--text-board {
+  color: #211952;
+  font-size: 18px;
+  letter-spacing: 0.4px;
+  line-height: 26px;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+.wizard--text-list {
+  color: rgba(33,25,82,0.56);
+  font-size: 13px;
+  line-height: 1px;
+  margin-left: 5px;
+}
+.wizard--text-board-container {
+  margin-left: 0px;
+  margin-bottom: 0px;
+  text-align: left;
+}
+.wizard--container {
+  width: 30%;
+  margin: auto;
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+.wizard--container-inner {
+  width: 100%;
+}
+.wizard--container-center {
+  width: 100%;
+  justify-content: center;
+}
+.wizard--board {
+  height: 84px;
+  width: 162px;
+  border: 1px solid rgba(33,25,82,0.24);
+  border-radius: 8px;
+  background-color: #FFFFFF;
+  box-shadow: 0 9px 11px 0 rgba(85,57,255,0.06);
+  margin-top: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  overflow: auto;
+}
+.wizard--board-selected {
+  background-color: #FFD6EA;
+}
+.wizard--button-container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  margin-top: 10px;
+}
+.button {
+  height: 56px;
+  width: 200px;
+  border: 1px solid #211952;
+  background-color: #FFFFFF;
+  margin: 5px;
+  color: #211952;
+  font-family: Rubik;
+  font-size: 20px;
+  font-weight: 500;
+  letter-spacing: 0.4px;
+  line-height: 24px;
+}
+.button-save {
+  background-color: #FCEA10;
+  border: none;
+}
+.wizard--container-list {
+  margin-top: 10px;
+  height: 484px;
+  width: 60%;
+  border: 1px solid rgba(255,214,234,0.57);
+  border-radius: 8px;
+  box-shadow: 0 9px 11px 0 rgba(85,57,255,0.06);
+  display: block;
+  overflow: auto;
+}
+.wizard--container-list-special {
+  width: 95%;
+}
+.checkbox-container {
+  width: max-content;
+  height: 42px;
+  border-radius: 21px;
+  background-color: #FFFFFF;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 8px;
+  z-index: 0;
+}
+.checkbox-container-selected {
+  background-color: rgba(92, 234, 165, 0.08);
+}
+.checkbox {
+  height: 20px;
+  width: 20px;
+  display: inline-block;
+  background-color: #F0F0F0;
+  z-index: 1;
+}
+.checkbox-container input[type=checkbox] {
+  display: none;
+}
+input[type=checkbox]:checked + .checkbox {
+  border-radius: 10px;
+  background-color: #5CEAA5;
+}
+.checkbox-container input[type=radio] {
+  display: none;
+}
+input[type=radio]:checked + .checkbox {
+  border-radius: 10px;
+  background-color: #5CEAA5;
+}
+.wizard--container-wide {
+  width: 80%;
+}
```

Comment:
Unexpected missing end-of-source newline                    no-missing-end-of-source-newline